bitnami/clickhouse/values.yaml

```diff
@@ -260,6 +260,8 @@ defaultConfigurationOverrides: |
             <replica>
                 <host>{{ printf "%s-shard%d-%d.%s.%s.svc.%s" (include "common.names.fullname" $ ) $shard $i (include "clickhouse.headlessServiceName" $) (include "common.names.namespace" $) $.Values.clusterDomain }}</host>
                 <port>{{ $.Values.service.ports.tcp }}</port>
+                <user>{{ $.Values.auth.username }}</user>
+                <password>{{ $.Values.auth.password }}</password>
```

Comment:
Could you please make it dependant of an env var? Otherwise the password will be plain

Reply:
Yes. I will create a new PR.